tests/direct_scraper/test_direct_scraper.py

```diff
@@ -40,6 +40,17 @@ def test_url_download(self):
         self.assertTrue(os.path.isfile(os.path.join(self.temp_dir,
                                                     scraper.target)))
 
+        scraper2 = DirectScraper(url=test_url,
+                                destination=os.path.join(self.temp_dir, 'file.html'),
+                                version=None,
+                                log_level='ERROR')
+        self.assertEqual(scraper2.target,
+                         os.path.join(self.temp_dir, 'file.html'))
+
+        scraper2.download()
+        self.assertTrue(os.path.isfile(os.path.join(self.temp_dir,
+                                                    scraper2.target)))
```

Comment:
It will be enough to add a test for the base scraper in case of a filename as destination because the logic is contained only there.